packages/babel-helpers/src/helpers.js

```diff
@@ -1299,24 +1299,90 @@ helpers.classPrivateFieldLooseBase = helper("7.0.0-beta.0")`
 `;
 
 helpers.classPrivateFieldGet = helper("7.0.0-beta.0")`
```

Comment:
We don't need to bump the version because it's functionally equivalent, right?

Reply:
Yes. Refactoring internals helpers should not bump minimal required helper version as long as the interface is not changed.